static/js/invite.js

```diff
@@ -115,11 +117,14 @@ function generate_multiuse_invite() {
         beforeSend,
         success(data) {
             ui_report.success(
-                i18n.t('Invitation link: <a href="__link__">__link__</a>', {
+                // id="link" for the a tag appended
+                i18n.t('Invitation link: <a href="__link__" id="link">__link__</a>', {
                     link: data.invite_link,
                 }),
                 invite_status,
             );
+            // function called here for copying
+            new ClipboardJS(".btn");
```

Comment:
This selector is way too wide in scope -- I think this will tag every button on the page?

Reply:
Should I replace it with any other name then?